### ISSUE TITLE
Link to Adjust's Privacy Policy is a 404

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#4649]
On https://accounts.firefox.com/legal/privacy we have this content:

{quote}On iOS and Android: Firefox by default sends mobile campaign data to Adjust, our analytics vendor, which has its own [privacy policy](https://www.adjust.com/privacy_policy/). Mobile campaign data includes a Google advertising ID, IP address, timestamp, country, language/locale, operating system, and app version. [...]{quote}

The privacy policy link goes to https://www.adjust.com/privacy_policy/ which is a 404.  It looks like the correct link would be https://www.adjust.com/terms/privacy-policy/

Should we adjust (pun intended!) our link or is it possible to reword this to remove the link altogether?

┆Issue is synchronized with this [Jira Bug](https://jira.mozilla.com/browse/FXA-1407)
┆Issue Number: FXA-1407

**Comment by apsychogirl** [comment]
Hi, Can I take this up?

**Comment by clouserw** [comment]
Hi @apsychogirl .  I assigned this to myself because it's a little more complex than it appears (the fix isn't even in this repository).  If you're looking for issues, check out the Help Wanted tag.  Cheers

**Comment by apsychogirl** [comment]
> Hi @apsychogirl . I assigned this to myself because it's a little more complex than it appears (the fix isn't even in this repository). If you're looking for issues, check out the Help Wanted tag. Cheers

Okay:)

**Comment by data-sync-user** [comment]
➤ Chelsea Lewis commented:

[~wclouser@mozilla.com] reviewed with Legal today, we just need to update the link to https://www.adjust.com/terms/privacy-policy/ instead of privacy_policy.

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

Pull request is open at https://github.com/mozilla/legal-docs/pull/1424

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

The pull request landed.  Due to https://jira.mozilla.com/browse/FXA-1023 we need to manually update our legal docs hash, which I've opened in https://github.com/mozilla/fxa/pull/4809

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

Fixed in https://github.com/mozilla/fxa/pull/4809
